src/app/customer/Customer.tsx

```diff
@@ -375,6 +413,22 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps, Cust
 
         onChangeViewType(CustomerViewType.Login);
     };
+
+    private executePaymentMethodCheckoutOrContinue: () => void = async () => {
+        const {
+            executePaymentMethodCheckout,
+            onContinueAsGuest = noop,
+            providerWithCustomCheckout,
+        } = this.props;
+
+        console.log('test');
```

Comment:
Left behind?

Reply:
Yeah, I forgot to remove it after debugging 😔

Reply:
Fixed